lib-external/jogl-gluegen/build-gluegen.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+
+cd gluegen/make
+ant -Dtarget.sourcelevel=1.8 -Dtarget.targetlevel=1.8 -Dtarget.rt.jar=/home/mpeterson/d/jopengl/dummy.jar
```

Comment:
Isn't there a problem with this referring to a local folder in your workstation?